torch/csrc/jit/ir.h

```diff
@@ -499,6 +499,15 @@ struct Node : public Attributes<Node> {
     }
   }
 
+  void replaceInputWithList(size_t i, ArrayRef<Value*> to) {
```

Comment:
I think this is a bit too specialized to be a method in ir.h, can we move it peephole.cpp